### PR TITLE
Add slightly tsundere Farsi translation.

### PR DESCRIPTION
**Pull request by Aidstastisch**

### CODE REVIEW

**Comment by Tillerino** [comment]
Awesome! Since a couple of things need to be corrected in the source code, I'll actually merge this one in by hand. I'll leave the pull request open for now and will write a comment when I start merging, so feel free to correct any mistakes that you might still find.

**Comment by Aidstastisch** [comment]
@Tillerino 
In the meantime I will ask people to review this translation as well.

File: tillerinobot/src/main/java/tillerino/tillerinobot/lang/Farsi.java [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+	}
+
+	@Override
+	public String noPercentageEstimates() {
+		return "Bebaghshid, wali endafe nemitoonam begam in chie.";
+	}
+
+	@Override
+	public void optionalCommentOnLanguage(IRCBotUser user, OsuApiUser apiUser) {
+		user.message("Pas modeli ke hastam doost daari, haan? :)");
```
Comment (Tillerino) [inline tillerinobot/src/main/java/tillerino/tillerinobot/lang/Farsi.java]: This should say something like "[your IGN] taught me how to speak Farsi."

**Comment by Aidstastisch** [comment]
@Tillerino I couldn't find a sort of "IGN", so I improvised.

**Comment by Aidstastisch** [comment]
@Tillerino 
Lol I thought IGN as in https://en.wikipedia.org/wiki/IGN

**Comment by Aidstastisch** [comment]
@Tillerino 
I and someone else took a look at the translation and I just committed our changes.

File: tillerinobot/src/main/java/tillerino/tillerinobot/lang/Farsi.java [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+	}
+
+	@Override
+	public String noPercentageEstimates() {
+		return "Bebaghshid, wali endafe nemitoonam begam in chie.";
+	}
+
+	@Override
+	public void optionalCommentOnLanguage(IRCBotUser user, OsuApiUser apiUser) {
+		user.message("ikhebepicmuis-jan learned me to talk Farsi!");
```
Comment (Tillerino) [inline tillerinobot/src/main/java/tillerino/tillerinobot/lang/Farsi.java]: I don't know if I was unclear or if you have a different opinion on this. I think this line should be in Farsi. Don't you think?

**Comment by Tillerino** [comment]
I manually merged this. Can you check if everything is working as expected?

**Comment by Tillerino** [comment]
Well, I guess it is...

**Pull request closed without merging**